wpengine/faustjs
# Fix CPT URL Rewrites

## Tasks


## Description

When the option "Enable Post and Category URL rewrites" is turned on currently there is no check for any custom post types. This feature will currently only work on the default pages & posts.

## Testing

1. Create a custom post type using ACF or similar that is public (Visible on the frontend and in the admin dashboard.)
2. Ensure the Faust option "Enable Post and Category URL rewrites" is enabled
3. From the `canary` branch, create a new post in that custom post type.
4. Once created, inspect the "view" link in the post list. This should match the WordPress URL
    ![Screenshot 2024-01-26 at 8 47 49 AM](https://github.com/wpengine/faustjs/assets/5946219/1144a254-f700-47fa-948a-ef72a6905e34)
5. Now, checkout this PR
6. Visit the same custom post type list again, and notice the view link is now using the headless site url
    ![Screenshot 2024-01-26 at 8 47 40 AM](https://github.com/wpengine/faustjs/assets/5946219/d1bb0f6c-4936-452f-b501-0937bf5a8be9)
